commands/ps/index.js

```diff
@@ -123,25 +103,26 @@ function printDynos (dynos) {
   })
 }
 
-const appquotaheader = {Accept: 'application/vnd.heroku+json; version=3.app-quotas'}
-
 function * run (context, heroku) {
   const {app, flags, args} = context
   const types = args
   const {json, extended} = flags
   const suffix = extended ? '?extended=true' : ''
 
-  let {quota, dynos, feature} = yield {
-    quota: heroku.post(`/apps/${app}/actions/get-quota${suffix}`, {headers: appquotaheader}).catch(() => {}),
-    dynos: heroku.request({path: `/apps/${app}/dynos${suffix}`}),
-    feature: heroku.get('/account/features/free-2016').catch(function (err) {
-      if (err.statusCode === 404 && err.body && err.body.id === 'not_found') {
-        return {enabled: false}
-      }
-      throw err
+  let promises = {
+    dynos: heroku.request({path: `/apps/${app}/dynos${suffix}`})
+  }
+
+  if (!json && !extended) {
+    promises.app_info = heroku.request({
+      path: `/apps/${context.app}`,
+      headers: {Accept: 'application/vnd.heroku+json; version=3.process-tier'}
     })
+    promises.account_info = heroku.request({path: '/account'})
   }
 
+  let {dynos, app_info, account_info} = yield promises
```

Comment:
@mnoble no, there are tests for this, plus I was unsure of this myself so I tested manually, they will be undefined